runtime/tokio_util.rs

```diff
@@ -20,5 +22,12 @@ where
 {
   let rt = create_basic_runtime();
   let local = tokio::task::LocalSet::new();
-  local.block_on(&rt, future)
+  let ret = local.block_on(&rt, future);
+
+  // Any call to `spawn_blocking` on the tokio runtime, which includes file I/O
+  // and non-blocking FFI, would usually keep the runtime from shutting down
+  // until they finish. With FFI this can keep the process from ever shutting
+  // down, so instead we set a maximum waiting time of half a second.
+  rt.shutdown_timeout(Duration::from_secs_f64(0.5));
```

Comment:
This will affect other things. For example: https://github.com/denoland/deno/pull/18401#discussion_r1149294420